addons/html_editor/static/tests/keyboard/arrow.test.js

```diff
@@ -390,6 +390,221 @@ describe("Around icons", () => {
+            contentAfterEdit: unformat(`
+                <p><br>
+                <span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span>[]
+                </p>
+                <p><br></p>
+            `),
+            contentAfter: unformat(`
+                <p><br>
+                <span class="fa fa-music"></span><span class="fa fa-music"></span>[]
+                </p>
+                <p><br></p>
+            `),
+        });
+        await testEditor({
+            contentBefore: unformat(`
+                <p><span class="fa fa-music"></span><span class="fa fa-music"></span>[]<br>
+                <span class="fa fa-music"></span><span class="fa fa-music"></span>
+                </p>
+            `),
+            contentBeforeEdit: unformat(`
+                <p><span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span>[]<br>
+                <span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span>
+                </p>
+            `),
+            stepFunction: async () => await press("ArrowDown"),
+            contentAfterEdit: unformat(`
+                <p><span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span><br>
+                <span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span>[]
+                </p>
+            `),
+            contentAfter: unformat(`
+                <p><span class="fa fa-music"></span><span class="fa fa-music"></span><br>
+                <span class="fa fa-music"></span><span class="fa fa-music"></span>[]
+                </p>
+            `),
+        });
+        await testEditor({
+            contentBefore: unformat(`
+                <p><span class="fa fa-music"></span>[]<span class="fa fa-music"></span><br>
+                <span class="fa fa-music"></span><span class="fa fa-music"></span>
+                </p>
+            `),
+            contentBeforeEdit: unformat(`
+                <p><span class="fa fa-music" contenteditable="false">\u200b</span>[]<span class="fa fa-music" contenteditable="false">\u200b</span><br>
+                <span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span>
+                </p>
+            `),
+            stepFunction: async () => await press("ArrowDown"),
+            contentAfterEdit: unformat(`
+                <p><span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span><br>
+                <span class="fa fa-music" contenteditable="false">\u200b</span><span class="fa fa-music" contenteditable="false">\u200b</span>[]
+                </p>
+            `),
```

Comment:
I think this is wrong. We were between the two icons on the first line, we should end up between the two icons on the second line.